src/shared/dtos/pagination-params.dto.ts

```diff
@@ -9,7 +9,7 @@ export class PaginationParamsDto {
   })
   @IsNumber()
   @IsOptional()
-  @Transform((value) => parseInt(value, 10), { toClassOnly: true })
+  @Transform(({ value }) => parseInt(value, 10), { toClassOnly: true })
```

Comment:
`{ value }`
Is this the new convention for passing values to Transform function? 
Will the input parameter as an object work?
Just curious to know.

Reply:
Yeah, this is the new way, we are destructuring the input and taking value only. Before it was only one param but now includes a lot of things.